documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR.
```

Comment:
Could you ask service team to follow https://aka.ms/azsdk/dpgcodegen to create the swagger PR?

Reply:
Is there any difference for the PR in public repo or private repo?

Reply:
I cannot add the link https://aka.ms/azsdk/dpgcodegen.
there is no difference for PR in private repo.